src/Apps/W1/Quality Management/app/src/Integration/Receiving/QltyReceivingIntegration.Codeunit.al

```diff
@@ -486,7 +486,7 @@ codeunit 20411 "Qlty. Receiving Integration"
     /// </summary>
     /// <param name="PurchaseLine">The purchase line</param>
     /// <param name="PurchaseHeader">The purchase header</param>
-    /// <param name="TempTrackingSpecification">Temporary VAR Record "Tracking Specification".</param>
+    /// <param name="TempTrackingSpecification">Temporary var Record "Tracking Specification".</param>
```

Comment:
This is not proper documentation. What is the parameters suppose to represent? Not the signature.